frontend/src/contexts/wishlist-context.tsx

```diff
@@ -0,0 +1,79 @@
+import { createContext, ReactNode, useState } from 'react';
+
+import { WishlistInput } from '@autoline/shared/common/types/types';
+import { HeartIcon } from '@components/common/icons/icons';
+import { Notification } from '@components/common/notification/notification';
+import {
+  useCreateWishlistMutation,
+  useDeleteWishlistMutation,
+  useGetWishlistStatusQuery,
+} from '@store/queries/preferences/wishlist';
+
+type WishlistContextType = {
+  likedCars: string[] | undefined;
```

Comment:
If the prop can be undefined, use `?:` type:
```suggestion
  likedCars?: string[];
```